src/status_im/chat/models.cljs

```diff
@@ -108,5 +108,4 @@
 
 (defn bot-only-chat? [db chat-id]
   (let [{:keys [group-chat contacts]} (get-in db [:chats chat-id])]
-    (and (not group-chat)
-         (get-in db [:contacts/contacts (:identity (first contacts)) :dapp?]))))
+    (and (not group-chat) (get-in db [:contacts/contacts (first contacts) :dapp?]))))
```

Comment:
is `(first contacts)` console chat?

Reply:
`:contacts` key in `chat` map contains just contacts identities, and yes, in case of "console" chats, it's just "console".